docs/content/concepts/partitions-schedules-sensors/partitions.mdx

```diff
@@ -133,6 +134,49 @@ result = materialize(
 )
 ```
 
+#### Dynamically partitioned assets (Experimental)
+
+<Note>
+  This feature is <strong>experimental</strong>. Until this feature is marked as
+  stable, partition created prior may not be retained.
+</Note>
+
+The <PyObject object="DynamicPartitionsDefinition" /> class accepts a name argument, representing the name of the partition set:
```

Comment:
Before jumping into the details of the classes, I think it's worth providing a sentence of explanation for why someone would want to use this feature.  E.g. "Sometimes you don't know the set of partitions ahead of time when you're defining your assets. For example, maybe you want to add a new partition every time a new data file lands in a directory. In these cases, you can use a `DynamicPartitionsDefinition`."

Reply:
this sounds great! thanks for the blurb